patch/osc133.c

```diff
@@ -0,0 +1,23 @@
+void scrolltoprompt(const Arg *arg) {
```

Comment:
This function implicitly depends on other patches and I need help in placing macro guards here as I'm not yet familiar with st's internals.